docs/section22/Rule22-42.md

```diff
@@ -0,0 +1,99 @@
+                    - otherwise this value doesn't exist, append these conditions to the missing_capacity_validation_points list: `else: missing_capacity_validation_points.append({"CHWT": cwt, "ECWT": ect})`
+
+            - the following lines or logic do the power validation check:
+            - create a list of non-matching power validation points: `non_matching_power_validation_points = []`
+            - create a list of missing power validation points: `missing_power_validation_points = []`
+            - loop through the expected_cwt_temps: `for cwt in expected_cwt_temps:`
+                - loop through each expected_ecwt_temps: `for ect in expected_ecwt_temps:`
+                    - look for the power validation points in power_validation_pts_dict: `if power_validation_pts_dict[[cwt, ect]]:`
+                        - we are expecting to see multiple validation points aligning with the expected_validation_plr.  Create a list of part load ratios that are given.  Later we'll compare this with the expected list to make sure that all points are given: `given_plrs = []`
+                        - look at each power validation point in the list: `for power_validation_point_result in power_validation_pts_dict[[cwt, ect]]:`
+                            - get the load: `load = power_validation_point.load`
+                            - get the given power: `given_power = power_validation_point_result`
+                            - calculate the PLR by dividing the load by the given capacity at these operating conditions: `plr = load / given_capacities[[cwt, ect]]`
+                            - check whether the plr is one of the plrs that we need to check: `if plr in expected_validation_plr:`
+                                - add the plr to the list of plrs provided: `given_plrs.append(plr)`
+                                - calculate eir_plr using the coefficients given: `eir_plr = plr_coefficients[0] + plr_coefficients[1] * plr + plr_coefficients[2] * plr^2`
+                                - calculate the eir_ft using the coefficients given: `eir_ft = eir_f_t_coefficients[0] + eir_f_t_coefficients[1] * cwt + eir_f_t_coefficients[2] * cwt^2 + eir_f_t_coefficients[3] * ect + eir_f_t_coefficients[4] * ect^2 + eir_f_t_coefficients[5] * cwt * ect
+                                - calculate the expected power using the formula:
+                                    - Chiller operating power = Rated Capacity × CAP-f-T × EIR-f-T × EIR-f-PLR × Chiller Input Power at Rated Conditions/Chiller Capacity at Rated Conditions
```

Comment:
Synax looks of in terms of ` use

Reply:
this is supposed to be non-code information